calyx-py/calyx/builder.py

```diff
@@ -192,6 +227,10 @@ def group(self, name: str, static_delay: Optional[int] = None) -> GroupBuilder:
 
     def comb_group(self, name: str) -> GroupBuilder:
         """Create a new combinational group with the given name."""
+        if isinstance(self.component, ast.CombComponent):
+            raise AttributeError(
+                "Sequential components do not have combinational groups, only wires."
```

Comment:
Should this say "combinational component"? (rather than sequential). Or do you mean something else by "sequential"?

Reply:
Thanks!